build/tasks/styles.js

```diff
@@ -212,3 +218,35 @@ function createPostCSSProcessor(pluginsListOrMap, options)
 
     return postcss(plugins);
 }
+
+/**
+ * Purge unused styles of minified CSS files.
+ *
+ * @async
+ *
+ * @param  {string} outfile  - The path of the CSS file.
+ * @param  {string} filestem - The CSS file name.
+ */
+async function purgeUnusedCSS(outfile, filestem) {
+
+    const timeLabel = `${filestem}.css purged in`;
+        console.time(timeLabel);
```

Comment:
Remove extraneous line break and fix indentation of console line.

```suggestion
    const timeLabel = `${filestem}.css purged in`;
    console.time(timeLabel);
```